pkg/deployment/deployment.go

```diff
@@ -101,10 +101,14 @@ func Pull(apiClient *clients.Settings, name, nsname string) (*Builder, error) {
 
 	if name == "" {
 		builder.errorMsg = "deployment 'name' cannot be empty"
+
+		return &builder, fmt.Errorf(builder.errorMsg)
 	}
 
 	if nsname == "" {
 		builder.errorMsg = "deployment 'namespace' cannot be empty"
+
+		return &builder, fmt.Errorf(builder.errorMsg)
```

Comment:
return &builder, fmt.Errorf("deployment 'namespace' cannot be empty")

Reply:
Done.